src/index.html

```diff
@@ -1,13 +1,508 @@
+                  <a
+                    href="https://www.facebook.com/"
+                    target="_blank"
+                    class="socials__icon-link"
+                  >
+                    <svg
+                      class="icon icon__facebook"
+                      width="24"
+                      height="24"
+                      aria-label="Фейсбук"
+                    >
+                      <use href="./images/icons/sprite.svg#facebook"></use>
+                    </svg>
+                  </a>
+                </li>
+                <li class="icon socials__icon">
+                  <a
+                href="https://www.instagram.com/"
+                target="_blank"
+                class="socials__icon-link"
+                  >
+                    <svg
+                      class="icon icon__instagram"
+                      width="24"
+                      height="24"
+                      aria-label="Інстаграм"
+                    >
+                      <use href="./images/icons/sprite.svg#instagram"></use>
+                    </svg>
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <div class="schedule footer__schedule">
+              <p class="schedule__title">Графік роботи</p>
+              <ul class="schedule__list">
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">12:00 — 20:00</span> середа
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">12:00 — 20:00</span> четвер
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">12:00 — 20:00</span> п’ятниця
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">11:00 — 19:00</span> субота
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">11:00 — 19:00</span> неділя
+                </li>
+              </ul>
+              <p class="schedule__weekend schedule__weekend--footer">
+                <span class="schedule__weekend--strong-footer">Вихідні</span> понеділок, вівторок<br> Каса припиняє роботу за 1
+                годину до закриття музею
+              </p>
+            </div>
+            <address class="contacts footer__contact">
+              <p class="contacts__title">Контакти:</p>
+              <ul class="contacts__list">
+                <li class="contacts__item">
+                  <a
+                    href="https://www.google.com.ua/maps/place/%D1%83%D0%BB.+%D0%9C%D0%B8%D1%85%D0%B0%D0%B8%D0%BB%D0%B0+%D0%93%D1%80%D1%83%D1%88%D0%B5%D0%B2%D1%81%D0%BA%D0%BE%D0%B3%D0%BE,+6,+%D0%9A%D0%B8%D0%B5%D0%B2,+02000/@50.4494431,30.5281844,17z/data=!3m1!4b1!4m6!3m5!1s0x40d4ce52445a8315:0x5e30ef474eddac89!8m2!3d50.4494397!4d30.5307593!16s%2Fg%2F11l76wtstx?entry=ttu"
```

Comment:
delete space

Reply:
I don't understand were you see the space, this tag doesn't have the space, it's ok

Reply:
Never mind it a GitHub review bug